api/app/controllers/webhook.controller.js

```diff
@@ -0,0 +1,48 @@
+const db = require("../sequelize/models");
+const User = db.user;
+const Review = db.review;
+const Transcript = db.transcript;
+const Op = db.Sequelize.Op;
+
+
+exports.create = async (req, res) => {
+    // const action = req.headers["x-github-event"];
+    const pull_request = req.body;
+     const action = pull_request.action;
+
+
+    // Check if it's a pull request event
+    if (action === 'opened' || action === 'closed' || action === 'synchronize') {
```

Comment:
We shouldn't hard-code values. We can create an enum or const to use in this case
```suggestion
const PR_EVENTS = {
opened: "opened",
closed: "closed",
sync: "synchronized",
} as const
    if (action === PR_EVENTS.opened || ......) {
```